### ISSUE TITLE
No VMnet interface available between vmnet2 and vmnet255 - GNS3 and Windows7 VM on VMware Fusion

### ISSUE THREAD

**Issue by grossmj** [GNS3/gns3-server#1912]
Community link: https://gns3.com/community/support/no-vmnet-interface-available-between-vmnet2-and-vmnet255-gns3-and-windows7-vm-on-vmware-fusion

**Comment by grossmj** [comment]
Please try this development version: https://sourceforge.net/projects/gns-3/files/Nightly%20Builds/2021-06-08/2.2/GNS3-2.2.22dev1.dmg/download

This should fix this issue. It works on my side but please confirm. Thanks :+1:

**Comment by grossmj** [comment]
I will consider it fixed. Please comment back or open a new issue if something does not work as expected. Thanks.

**Comment by jnatschev** [comment]
hello @grossmj .

i've done a preliminary validation and am happy to report that the issue has been fixed.

i will try things a bit more in depth and report back. i'll also review the code changes to try and get a better handle the fix.

apologies i wasn't able to try the development version in a more timely manner. other commitments.

this is a great outcome. 

so, thank you very very very much.

onwards and forwards,
john natschev.

**Comment by grossmj** [comment]
No worries, we had a tight deadline to release 2.2.22 :) Thanks a lot.

**Comment by jnatschev** [comment]
did a bit more testing and works a treat @grossmj. cheers again, john natschev.

**Comment by grossmj** [comment]
> did a bit more testing and works a treat @grossmj. cheers again, john natschev.

Perfect, great to hear. thanks :+1:

**Comment by niyidrums** [comment]
Thanks @grossmj and @jnatschev for your great help and devotion to GNS3 project, appreciate your time in making sure this issue is sorted, thankful for you both.
Just to be sure, should I go ahead to use the development version shared or I should wait for a new GNS3 software update version 2.2.23? As @jnatschev validated, the development version solves the problem, which I think will be an added bug fix in 2.2.23; can you please confirm this @grossmj so I know which version release to use... 

Appreciate again.

**Comment by grossmj** [comment]
No worries, version 2.2.22 which has been released yesterday will work :)

**Comment by niyidrums** [comment]
Thanks for reverting @grossmj and excellent work again. I will update my software now and let you know. Thanks.

**Comment by karltehgreat** [comment]
Hi! I came across this issue troubleshooting my own environment today. I'm trying to connect my VM to my existing lab in order to get access to a management webserver on one of the devices in my project. So far I've been able to run the gns3vmnet script and I can manually connect the VM (within WMware fusion) to the vmnet10 adapter. However I can only see vmnet2 and 3 in GNS3. I get the exact same error message trying to start my VM through GNS3. I tried setting the VM to use vmnet3 in both gns3 and vmware, starting the VM through fusion and setting static addresses on the two devices, but they are unable to communicate. I'm currently running MacOS Monterey, GNS3 2.2.31 and VMware Fusion 12.2.3. Could it be caused by a similar issue?
